### ISSUE TITLE
Up/down arrow buttons in NumberBox are not showing any effect on mouse hovering.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#1852]
**Describe the bug**

**Steps to reproduce the bug**

1. Turn on High contrast mode (Alt+Shift+PrtScn).
2. Launch XAML Control Gallery app.
3. Navigate to "NumberBox" in All controls and activate it.
4. Under the header "A NumberBox with a spin button", navigate to "Enter an integer" combo box.
5. Now try to change the integer using up/down arrows present over there.
6. Observe the arrows on hover

**Expected behavior**

Up/down arrow buttons should follow high contrast mode.

In **High Contrast Black** mode :  The "Up/down arrow buttons" should be highlighted with aqua blue color on hovering .
In **High Contrast White** mode : The "Up/down arrow buttons" should be highlighted with Indigo color on hovering.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/72565429-aab48b80-3866-11ea-9029-401eb860177b.png)

![image](https://user-images.githubusercontent.com/13142665/72565445-b6a04d80-3866-11ea-833b-e27fc1b5918a.png)

**Version Info**

Xaml Controls Gallery version 1.2.12.0

NuGet package version: 

**Additional context**

Issue copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/24649974)

**Comment by StephenLPeters** [comment]
@TeaP fyi

**Comment by chingucoding** [comment]
Should the border be highlighted or should the background change color (like in non high contrast). 

In the first case we run into the problem that the buttons themself do not have a border. The border that is visible is the border of the container, not the buttons. Thus on highlight, with the current layout we can not change the border highlighting.

**Comment by mdtauk** [comment]
How is the focus indicator used for the compact buttons, and could that be used for High Contrast modes?

**Comment by chingucoding** [comment]
The normal hover for compact spinbuttons is that the background gets lighter/darker.
